tests/testPhotoCal.py

```diff
@@ -114,7 +117,6 @@ def tearDown(self):
     def getAstrometrySolution(self, loglvl=Log.INFO):
         astromConfig = measAstrom.AstrometryTask.ConfigClass()
         astrom = measAstrom.AstrometryTask(config=astromConfig, refObjLoader=self.refObjLoader)
-        # use solve instead of run because the exposure has the wrong bbox
```

Comment:
The comment is dead wrong. `AstrometryTask.solve` does not support specifying a different bbox, so I made sure the bbox of the exposure was correct. There is no strong reason to prefer `solve` over `run`, but `run` is a thin layer that calls `solve` or `loadAndMatch`, depending on a config flag, and what I wanted was `solve`. In general I think it better to call `AstrometryTask.solve` or `loadAndMatch` rather than run, if you are sure what you want the task to do, and the task is documented as such.